source/_integrations/reolink.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Reolink NVR/camera
+description: Instructions on how to integrate Reolink devices (NVR/cameras) into Home Assistant.
+ha_category:
+  - Camera
+ha_iot_class: Local Polling
+ha_release: 2023.1
+ha_domain: reolink
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+ha_platforms:
+  - Camera
+ha_integration_type: integration
+---
+
+The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
+
+{% include integrations/config_flow.md %}
+
+## Known reolink firmware bugs
+
+<div class='note warning'>
+There is a BUG in Reolink NVR firmware: it only sends ONVIF event-notifications if motion happened on the camera connected to its very first (index "0") channel.
+Reolink is aware of this and are working on fixing the problem.
+</div>
+
+<div class='note warning'>
+Reolink doorbell presses only generate ONVIF event-notifications when the doorbell is directly connected to your network.
+When connecting the Reolink doorbell to a NVR, no doorbell press events are generated.
+Reolink is aware of this and a solution is already implemented in beta firmware.
+</div>
+
+## Camera streams
+
+This integration creates a few camera entities, one for each stream type with different resolutions: Main, Sub, Ext and Images.
+The Sub stream camera entity is enabled by default, the other streams are disabled by default.
+The Images stream provides a sequence of image snapshots giving very low latency at the cost of a very low frame rate, this can be used when the hi-res RTMP/RTSP video stream are too laggy.
+
+## Configuration options
+
+In the options menu, the following parameters can be configured:
+
+| Parameter               | Description                                                                                                 |
+| :-------------------    | :---------------------------------------------------------------------------------------------------------- |
+| Protocol                | Switch between the RTMP or RTSP streaming protocol.                                                         |
+
+## Unsupported models
+
+The following models are not to be supported:
+
+- Battery-powered cameras
+- B800: Only with NVR
+- B400: Only with NVR
+- D400: Only with NVR
+- Lumus
```

Comment:
It would be better to list the tested models instead of the unsupported ones since usually people want to buy hardware that based on what is supported and not what they know won't work

Reply:
I updated it to list the models I have tested.
I am sure that it works with a lot more models, but I do not have a list of which are tested and not.
I left the note about battery-powered cameras, because they will not be supported for a while (not always available for connection).